src/data/encodeQueryString.ts

```diff
@@ -18,7 +19,7 @@ export const encodeQueryString = ({
 
   // Iterate params, the keys are prefixed with `$` and their values JSON stringified
   for (const [key, value] of Object.entries(params)) {
-    searchParams.append(`$${key}`, JSON.stringify(value))
+    if (isDefined(value)) searchParams.append(`$${key}`, JSON.stringify(value))
```

Comment:
I don't have a lot of context on the client so I will leave it to other folks to weight in, however, this might be considered a breaking change for other devs that might have been using the ability to send in nullable values.

Reply:
You are right, @RitaDias. The code now only prevents `undefined` values from being passed to search parameters.